### ISSUE TITLE
Mosaic giving orange-yellow color to the result

### ISSUE THREAD

**Issue by eyalmazuz** [deeppomf/DeepCreamPy#198]
**Describe the bug**
When using the mosaic option with the new version 2.2
I get orange-yellow lava-like results on the mosaic parts
something that looks like it came from hell itself.

**To Reproduce**

1. Follow the guide and use color RGB(0, 255, 0)
2. Run python main.py launching the gui version
3. Output image has yellow filter applied to working area (but nothing else appears to change)

**Expected behavior**
Idk...I expected good results I guess

**Installation method**
I'm using Linux so I ran from the source code

**Desktop (please complete the following information):**
OS: ubuntu 19.10
Python: 3.7.5
Tensorflow: 1.14
no AVX

**Comment by MellowEdge** [comment]
Yeah, I'm having the same problem, but I don't want to revert to an older, probably worse model. Did some commit bring back older code or something?

**Comment by eyalmazuz** [comment]
> Yeah, I'm having the same problem, but I don't want to revert to an older, probably worse model. Did some commit bring back older code or something?

Yes, older code, there is no graphical user interface and probably different model.

**Comment by MellowEdge** [comment]
It's gotta be something with https://github.com/deeppomf/DeepCreamPy/commit/f5a1b34066c4b2e2a7c2f2a40a89744846d1d50c and/or 2 commits before it, since on commit https://github.com/deeppomf/DeepCreamPy/commit/17c5c4cadd9727f9153091c0d107f6560cb6fd93 it still works fine.

**Comment by piespy** [comment]
The problem is that with the new GUI, the model is loaded at startup. However, it's always loading the bar censor model by default. If you select mosaic censor in the GUI, it will still use the previously loaded model.
So the workaround for now is to add "--is_mosaic true" to the command line like in the old version and ignore the GUI censor type selection for now.
The real solution would be to load the other model whenever the censor type selection is changed, or load both models at startup and switch between them, but either way would take a longer time than using the right command line option.

**Comment by deeppomf** [comment]
@piespy Don't know how that bug escaped my notice.
Will be fixed soon.
